theme/themes/pastanaga/extras/blocks.less

```diff
@@ -304,7 +304,7 @@ body.has-toolbar.has-sidebar-collapsed .ui.wrapper > .ui.inner.block.full {
 
   .toolbar-inner {
     display: flex;
-    width: 450px;
+    width: min(100%, 450px);
```

Comment:
@sneridagh The issue reproduce also on Volto core. This may help https://stackoverflow.com/questions/64255505/less-js-error-evaluating-function-min-incompatible-types